JavaImportTool/src/javaImportTool/ASTConverter.java

```diff
@@ -654,17 +653,19 @@ public void processTypeParameters(List<TypeParameter> params) throws ConversionE
 		}
 	}
 	
-	public void processParameters(List<SingleVariableDeclaration> params) throws ConversionException
+	public void processParameters(List<VariableDeclaration> params) throws ConversionException
 	{
 		int i = 0;
-		for(SingleVariableDeclaration arg : params)
+		for(VariableDeclaration arg : params)
 		{
 			Node a = containers.peek().child("arguments").add(new Node(null,"FormalArgument", i++));
 			a.setSymbol(arg.getName().getIdentifier());
 			setModifiersAndAnnotations(a, Modifier.PUBLIC, null);
 			
-			Node type = typeExpression(arg.getType(), "typeExpression");
-			a.setChild("typeExpression", addExtraDimensions(type,arg.getExtraDimensions()));
+			if (arg instanceof SingleVariableDeclaration) {
```

Comment:
True but I don't see how we can easily solve this: the parent node will be just any ASTNode, not sure how we would get the type from it.

On the other hand I doubt it can ever be the second case in this method, since we process parameters to a method or a lambda here.